Watchman.DomainModel/Settings/Configuration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings
+{
+    public class Configuration : Entity, IAggregateRoot
+    {
+        public int MinAverageMessagesPerWeek { get; private set; }
+
+        internal static Configuration GetDefaultConfiguration()
```

Comment:
lepiej internat static Default => return new Configuration { reszta kodu }

wtedy użycie to Configuration.Default;
myśle że wygląda to dużo estetyczniej